.github/workflows/testing.yml

```diff
@@ -0,0 +1,43 @@
+name: Testing
+
+on: [push]
+
+jobs:
+  code-style:
```

Comment:
There is a pre-commit action to do this, which should be faster
https://github.com/torchbox/wagtailmedia/blob/main/.github/workflows/test.yml#L15-L27

or, even better https://pre-commit.ci/ - but this is probably for another day

Reply:
OK thanks will take a look.

Reply:
This is updated